Branch-SDK/Branch-SDK/BranchUniversalObject.m

```diff
@@ -13,6 +13,153 @@
 #import "BNCDeviceInfo.h"
 #import "BNCLog.h"
 #import "BNCLocalization.h"
+#import "BNCEncodingUtils.h"
+#import "Branch.h"
+
+#pragma mark BranchContentSchema
+
+BranchContentSchema _Nonnull BranchContentSchemaCommerceAuction     = @"COMMERCE_AUCTION";
+BranchContentSchema _Nonnull BranchContentSchemaCommerceBusiness    = @"COMMERCE_BUSINESS";
+BranchContentSchema _Nonnull BranchContentSchemaCommerceOther       = @"COMMERCE_OTHER";
+BranchContentSchema _Nonnull BranchContentSchemaCommerceProduct     = @"COMMERCE_PRODUCT";
+BranchContentSchema _Nonnull BranchContentSchemaCommerceRestaurant  = @"COMMERCE_RESTAURANT";
+BranchContentSchema _Nonnull BranchContentSchemaCommerceService     = @"COMMERCE_SERVICE";
+BranchContentSchema _Nonnull BranchContentSchemaCommerceTravelFlight= @"COMMERCE_TRAVEL_FLIGHT";
+BranchContentSchema _Nonnull BranchContentSchemaCommerceTravelHotel = @"COMMERCE_TRAVEL_HOTEL";
+BranchContentSchema _Nonnull BranchContentSchemaCommerceTravelOther = @"COMMERCE_TRAVEL_OTHER";
+BranchContentSchema _Nonnull BranchContentSchemaGameState           = @"GAME_STATE";
+BranchContentSchema _Nonnull BranchContentSchemaMediaImage          = @"MEDIA_IMAGE";
+BranchContentSchema _Nonnull BranchContentSchemaMediaMixed          = @"MEDIA_MIXED";
+BranchContentSchema _Nonnull BranchContentSchemaMediaMusic          = @"MEDIA_MUSIC";
+BranchContentSchema _Nonnull BranchContentSchemaMediaOther          = @"MEDIA_OTHER";
+BranchContentSchema _Nonnull BranchContentSchemaMediaVideo          = @"MEDIA_VIDEO";
+BranchContentSchema _Nonnull BranchContentSchemaOther               = @"OTHER";
+BranchContentSchema _Nonnull BranchContentSchemaTextArticle         = @"TEXT_ARTICLE";
+BranchContentSchema _Nonnull BranchContentSchemaTextBlog            = @"TEXT_BLOG";
+BranchContentSchema _Nonnull BranchContentSchemaTextOther           = @"TEXT_OTHER";
+BranchContentSchema _Nonnull BranchContentSchemaTextRecipe          = @"TEXT_RECIPE";
+BranchContentSchema _Nonnull BranchContentSchemaTextReview          = @"TEXT_REVIEW";
+BranchContentSchema _Nonnull BranchContentSchemaTextSearchResults   = @"TEXT_SEARCH_RESULTS";
+BranchContentSchema _Nonnull BranchContentSchemaTextStory           = @"TEXT_STORY";
+BranchContentSchema _Nonnull BranchContentSchemaTextTechnicalDoc    = @"TEXT_TECHNICAL_DOC";
+
+#pragma mark - BranchProductCondition
+
+BranchProductCondition _Nonnull BranchProductConditionOther         = @"OTHER";
+BranchProductCondition _Nonnull BranchProductConditionNew           = @"NEW";
+BranchProductCondition _Nonnull BranchProductConditionGood          = @"GOOD";
+BranchProductCondition _Nonnull BranchProductConditionFair          = @"FAIR";
+BranchProductCondition _Nonnull BranchProductConditionPoor          = @"POOR";
+BranchProductCondition _Nonnull BranchProductConditionUsed          = @"USED";
+BranchProductCondition _Nonnull BranchProductConditionRefurbished   = @"REFURBISHED";
+
```

Comment:
missing "EXCELLENT"

Reply:
Rename `eventDescription`:  `description` is a reserved word in objc, so no.

Reply:
`EXCELLENT` added.

Reply:
Renamed indexPublicly and indexLocally to locallyIndex and publiclyIndex.

Reply:
Changed `ratingMaximum` to `ratingMax`.

Reply:
* The field `creationDate` is existing, it's called `$creation_timestamp`.